docs/index.bs

```diff
@@ -3228,9 +3228,11 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Let |shouldSoftUpdate| be true if any of the following are true, and false otherwise:
           * |request| is a [=non-subresource request=].
           * |request| is a [=subresource request=] and |registration| is [=stale=].
+      1. Let |raceResponse| be null.
```

Comment:
I guess |raceResponse| can go to the line before L3293 to make its lifetime shorter?
We do not need to share |raceResponse| between "race-network-and-fetch-event" and SWAutpPreload.

Reply:
That's true. Moved this to L3319.

Reply:
Ah no, we have to pass `raceResponse` in L3328.

> 1. Let |fetchResult| be the result of [=Create Fetch Event and Dispatch=] with |request|, |registration|, |useHighResPerformanceTimers|, |timingInfo|, |workerRealm|, |reservedClient|, |preloadResponse|, and |raceResponse|.

So I think the current position is reasonable?

Reply:
Discussed offline, we don't have to use the same variable name.